index.html

```diff
@@ -177,7 +177,7 @@ <h2>Portfolio Heading</h2>
       <div class="row">
         <div class="col-lg-4 col-sm-6 portfolio-item">
           <div class="card h-100">
-            <a href="#"><img class="card-img-top" src="http://placehold.it/700x400" alt=""></a>
+            <a href="#"><img class="card-img-top img-fluid" src="http://placehold.it/700x400" alt=""></a>
```

Comment:
Glad you added this class, the responsive images will definitely help to improve the user experience.